src/shared/reducers/reducer.js

```diff
@@ -26,6 +26,8 @@ const toByIdMap = objects => objects.reduce((map, x) => {
   return map
 }, {})
 
+const setEventsCalendarId = calendarId => event => Object.assign({}, event, {calendarId})
```

Comment:
it's babel transpiled, so you can use `...`

Reply:
done